src/verifiers/documentStatus/tokenRegistry/ethereumPolTokenRegistryStatus.test.ts

```
@@ -0,0 +1,82 @@
+import { documentPolValidWithToken } from "../../../../test/fixtures/v2/documentPolValidWithToken";
+import { openAttestationEthereumTokenRegistryStatus } from "./ethereumTokenRegistryStatus";
+import { generateProvider, getDefaultProvider } from "../../../common/utils";
+import { verificationBuilder, openAttestationVerifiers, isValid } from "../../../index";
+
+// Polygon mainnet public RPC — no API key required for read-only calls.
+const POL_RPC_URL =
+  process.env.POL_RPC || "https://polygon-rpc.com";
```

ESLint

[error] Delete `⏎·`

(prettier/prettier)